Vignettes/usingcchsflow.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "Transform CCHS variables"
+date: 'Last updated: 2019-08-27'
+output: rmarkdown::html_vignette
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Getting Started
+### Loading the necessary packages
+
+The `RecWTable` and `SetDataLabels` functions are part of the [bllflow](https://bllflow.projectbiglife.ca/index.html) package. `RecWTable` is used to transform CCHS variables to a common name, while `SetDataLabels` reapplies labels that are lost during the combination process after transforming each CCHS dataset. You must install & load the package onto your environment in order to do any transformations. To be able to combine datasets with minimal issues, we recommend using the `bind_rows` function in the [dplyr](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8) package. 
+
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(bllflow)
```

Comment:
bllflow isn’t on CRAN. Provide instructions on how to install bllflow from GitHub.

Reply:
We many want to identify that bllflow is underdevelopment and that users may want to use the development branch. @DarkShadeKnigh ?

Reply:
That can be risky since we would have virtually no control over what they use. I updated the PR of the recode-table to master. However weve had format changes along the way so i need to update some older functions but the pr will be up soon and yulric said he has time to review it.

Reply:
I do agree about installing the github version